### ISSUE TITLE
New Config Parameter: Daily events - free vs. busy

### ISSUE THREAD

**Issue by upD8R** [derekantrican/GAS-ICS-Sync#211]
HI there,

this is an enhancement proposal - maybe you can add an option to overwrite how daily (whole day) events should be stored in GCal. In most cases I don't want to see them as "busy/booked" but as "free". Not sure if this can be achieved, though.

Thanks!

**Comment by derekantrican** [comment]
I see. We have an "overrideVisibility" setting, but not an "overrideAvailability" (sometimes called "transparency") setting. Is that what you are suggesting?

I think we can justify adding a global setting, but making it *just for all-day events* might be a little too niche

**Comment by upD8R** [comment]
Thanks for the feedback, Derek. I wonder if I maybe missed the overrideVisibility setting?
Let me explain where my request comes from: I use TickTick for task planning and they issue an ICS URL, which can be subscribed to GCal. I replaced that with your script to have a better handling on changes.

What I can see is that daily events are shown as busy, so it's hard/impossible to plan other appointments on that day because you don't see free slots (e.g. after subscribing to the same GCal calendar in e.g. Outlook).

It's not an urgent topic, though. I found a setup, which works for me ...
